src/components/date.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+/**
+ * Interface for the DateStr prop
+ */
+interface DateStr {
```

Comment:
I was imagining calling the component itself `DateStr`, then this interface would be called `DateStrProps`.